public/js/movieDetails.js

```diff
@@ -0,0 +1,200 @@
+
+function createElement(tag, attributes = {}, text = '') {
+	const element = document.createElement(tag);
+	Object.entries(attributes).forEach(([key, value]) => element.setAttribute(key, value));
+	if (text) element.textContent = text;
+	return element;
+}
+
+function detailsSection() {
+	const detailSection = createElement('section', { class: 'details-section' });
+	const movieInfo = createElement('div', { class: 'movie-info' });
+	const movieDiv = createElement('div', { class: 'movie-image' });
+	const movieImage = createElement('img', { src: 'images/thelastbreath.jpg', alt: 'A Quiet Place download' });
+
+	const movieDetails = createElement('div', { class: 'movie-details' });
+	const titleElement = createElement('h1', {}, movies[0].title);
+	const yearElement = createElement('p', {}, `Year: ${movies[0].year}`);
+	const genresElement = createElement('p', {}, `Genres: ${movies[0].genres}`);
+	const likesElement = createElement('p', {}, `Likes: ${movies[0].likes}`);
+	const ratingsTitle = createElement('h2', {}, 'Official Ratings');
+	const rottenTomatoes = createElement('p', {}, 'Rotten Tomatoes');
+	const imdb = createElement('p', {}, 'IMDB');
+
+	movieDetails.append(titleElement, yearElement, genresElement, likesElement, ratingsTitle, rottenTomatoes, imdb);
+
+	const similarMoviesContainer = createSimilarMovies();
+	movieDiv.appendChild(createElement('figure').appendChild(movieImage));
+	movieInfo.append(movieDiv, movieDetails, similarMoviesContainer);
+	detailSection.appendChild(movieInfo);
+
+	return detailSection;
+}
+
+function createSimilarMovies() {
+	const similarMoviesContainer = createElement('div', { class: 'similar-movies' });
+	const similarMoviesTitle = createElement('h1', {}, 'Similar movies');
+	const similarMoviesBox = createElement('div', { class: 'similar-movies-box' });
+	const imagesContainer = createElement('div');
+
+	movies.slice(0, 4).forEach(movie => {
+		const imageAnchor = createElement('a', { href: 'details' });
+		const movieImage = createElement(
+			'img',
+			{
+				src: `https://image.tmdb.org/t/p/w200//${movie.poster}`,
+				alt: `${movie.title} download`
+			}
+		);
+		imageAnchor.appendChild(movieImage);
+		imagesContainer.appendChild(imageAnchor);
+	});
+
+	similarMoviesBox.appendChild(imagesContainer);
+	similarMoviesContainer.append(similarMoviesTitle, similarMoviesBox);
+	return similarMoviesContainer;
+}
+
+
+function trailerSection() {
+	const trailerSection = createElement('section', { class: 'trailer-section' });
+	const trailerRow = createElement('div', { class: 'trailer-row' });
+
+	const movieTrailer = createElement('div', { class: 'movie-trailer' });
+	const video = createElement('video', { controls: 'controls' });
+	const movieSource = createElement('source', { src: 'video.mp4', type: 'video/mp4' });
+	video.append(movieSource, document.createTextNode('Your browser does not support the video tag.'));
+	movieTrailer.appendChild(video);
+
+	const thumbnails = Array(2).fill().map(() => {
```

Comment:
Abstain from using magic numbers since it is hard for people reading your code to understand what the numbers means at a glance